README.md

```diff
@@ -9,6 +9,7 @@ Web example is published from `gh-pages` branch on this url [https://tgrospic.gi
 Install project dependencies (in `./node_modules` folder).
 
 ```sh
+npm i @tgrospic/rnode-grpc-js
```

Comment:
@jimscarver thanks for the PR! :smile: 

You are right I didn't include recipe how to build this example from scratch. To start with the example you don't need to add `rnode-grpc-js` because it's specified in [package.json file](https://github.com/tgrospic/rnode-client-js/blob/1b2120a5243f38069459c6c9f8afd691b55da6a2/package.json#L21).

Installation instructions are specified in [rnode-grpc-js/README.md](https://github.com/tgrospic/rnode-grpc-js#install) but I think it would be useful to repeated it here or/and with the link to _rnode-grpc-js_. It should be clear that for running this example you don't need to do any of these steps.